### ISSUE TITLE
isilon-hadoop-tools, password and group file

### ISSUE THREAD

**Issue by radya13** [claudiofahey/isilon-hadoop-tools#20]
Hi every body,

I m trying to tie together isilon and hortonworks, using this guide https://www.emc.com/collateral/TechnicalDocument/docu71396.pdf

when i execute the code create_users.sh i dont see where does it put passwd and group files, ,'cause when i execute this command " cat <accesszone>.passwd " it doesnt find anything, i checked, but no way and this leades to deployment failure,

thank you very much.

**Comment by dmtucker** [comment]
@radya13 Hello, apologies for any confusion, but this repository has been replaced by [Isilon/isilon_hadoop_tools](https://github.com/Isilon/isilon_hadoop_tools).

To answer your question, [the `passwd` and `group` files will be named after the zone you initialized and placed in $PWD](https://github.com/Isilon/isilon_hadoop_tools/blob/c92e1907b9b67c8cdc8fd0529c1a4de94065a305/isilon_create_users.sh#L207) (e.g. `./System.passwd`). You should be able to view them (after running `isilon_create_users.sh`) with:
``` sh
cat ./*.passwd
cat ./*.group
```

**Comment by radya13** [comment]
Hi claudio . 
Thank you for ur answer . Nice of you , unfortunately i m still having the same problem. Here u can find the cli output

Kindly 
![capture d ecran 2017-04-10 a 20 42 56](https://cloud.githubusercontent.com/assets/21688491/24894878/1800243c-1e85-11e7-9566-2e4d7447ae43.png)
![capture d ecran 2017-04-10 a 20 42 56](https://cloud.githubusercontent.com/assets/21688491/24894887/2486f91a-1e85-11e7-88aa-1866d69f3e1c.png)

**Comment by dmtucker** [comment]
@radya13 Interesting... I suspect you are using an outdated version of the script. Output from [the latest version](https://github.com/Isilon/isilon_hadoop_tools/blob/master/isilon_create_users.sh) looks like this:
```
dtucker-users-1# curl 'https://raw.githubusercontent.com/Isilon/isilon_hadoop_tools/master/isilon_create_users.sh' > isilon_create_users.sh
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100 11593  100 11593    0     0  28089      0 --:--:-- --:--:-- --:--:-- 41256
dtucker-users-1# chmod +x isilon_create_users.sh
```
```
dtucker-users-1# ./isilon_create_users.sh --dist hwx --startgid 501 --startuid 501 --zone System
Info: Hadoop distribution:  hwx
Info: groups will start at GID 501
Info: users will start at UID 501
Info: will put users in zone:  System
Info: HDFS root:  /ifs/hdfs
Info: passwd file: System.passwd
Info: group file: System.group
SUCCESS -- Hadoop users created successfully!
Done!
```

**Comment by radya13** [comment]
Yes you are right i got the right scripts now and it works good . Thank you
so.much :*
